src/nwb2bids/sanitization/_levels.py

```diff
@@ -0,0 +1,26 @@
+import enum
+
+
+class SanitizationLevel(enum.IntEnum):
```

Comment:
Note unlike those other enum cases, I do actually want these values hard-coded and instead the associated names are just to add a bit of a more verbose reference